google/cloud/forseti/common/util/string_formats.py

```diff
@@ -14,14 +14,34 @@
 
 """Common formatting methods."""
 
+# Filename patterns.
+VIOLATION_JSON_FMT = 'violations.{}.{}.{}.json'
+SCANNER_OUTPUT_CSV_FMT = 'scanner_output_base.{}.csv'
+FINDINGS_FILENAME = 'forseti_findings_{}.json'
+
+# Timestamps.
 # Example: '2018-03-01T21:31:52'
 TIMESTAMP_UTC_OFFSET = '%Y-%m-%dT%H:%M:%S%z'
 
 # Example: '2018-03-01T21:32:24.491644'
 TIMESTAMP_MICROS = '%Y-%m-%dT%H:%M:%S.%f'
 
 # Example: '2018-03-01T21:33:59Z'
-TIMESTAMP_TIMEZONE_NAME = '%Y-%m-%dT%H:%M:%SZ'
+TIMESTAMP_TIMEZONE = '%Y-%m-%dT%H:%M:%SZ'
 
 # Example: '01 March 2018 - 21:38:12'
-TIMESTAMP_HUMAN_READABLE = '%d %B %Y - %H:%M:%S'
+TIMESTAMP_READABLE = '%d %B %Y - %H:%M:%S'
+
+# Example: '2018-03-01 23:25:59'
+TIMESTAMP_MYSQL_DATETIME_FORMAT = '%Y-%m-%d %H:%M:%S'
+
+# Example: '2018 Mar 01, 23:14:55 (UTC)'
+TIMESTAMP_READABLE_UTC = '%Y %b %d, %H:%M:%S (UTC)'
+
+# Example: '20180301T213359Z'
+TIMESTAMP_TIMEZONE_FILES = '%Y%m%dT%H%M%SZ'
+
+# Defaults to use globally.
+DEFAULT_FORSETI_TIMESTAMP = TIMESTAMP_TIMEZONE
```

Comment:
With default string formats we can insist in the code that when people need a string formatted datetime they should use the default (consistency for readers of or logs and/or data) as it's better for our users unless there is a good reason.

Before this I counted each scanner, notifier, etc using upwards of 5 different string formats of datetime for no necessary reason.

Reply:
Sorry, let me clarify.  I was asking about why do we need this specific `DEFAULT_FORSETI_TIMESTAMP = TIMESTAMP_TIMEZONE`.  :)  Why not just drop `TIMESTAMP_TIMEZONE`, so that you just have `DEFAULT_FORSETI_TIMESTAMP = '%Y-%m-%dT%H:%M:%SZ'` and use that everywhere?

Reply:
I opted for this because it gave flexibility to change things behind the scenes. It's also super clear what the default is when you look at this file because of the name in the assignment. If it were just a string you'd have no idea what the string resulted in.

Reply:
okay